content/blog/2019-10-02-dev-guide-update-fall19.md

```diff
@@ -0,0 +1,94 @@
+---
+slug: dev-guide-update-fall19
+title: 'rOpenSci Dev Guide 0.3.0: Updates Inside and Out'
+date: 2019-10-02
+authors:
+  - Scott Chamberlain
+  - Brooke Anderson
+  - Anna Krystalli
+  - Lincoln Mullen
+  - Karthik Ram
+  - Noam Ross
+  - Maëlle Salmon
+  - Melina Vidoni
+categories: blog
+topicid:
+tags:
+  - Software Peer Review
+---
+
+As [announced in February](/blog/2019/02/01/software-review-news/) updates to our Software Peer Review system, all our package development, review and maintenance is available as [an online book](https://devguide.ropensci.org/). Our goal is to update it approximately quarterly - it's time to present the third version. You can [read the changelog](https://devguide.ropensci.org/booknews.html) or this blog post to find out what's new in our dev guide 0.3.0!
```

Comment:
The 1st sentence is a bit unclear, there could be one "," more, and should it be "are available"?

Reply:
fixed